src/libmeasurement_kit/engine/task.hpp

```diff
@@ -4,81 +4,21 @@
 #ifndef SRC_LIBMEASUREMENT_KIT_ENGINE_TASK_HPP
 #define SRC_LIBMEASUREMENT_KIT_ENGINE_TASK_HPP
 
```

Comment:
The code removed from here was actually moved to `include/measurement_kit/internal/engine/task.hpp`